Dockerfile.centos

```diff
@@ -0,0 +1,49 @@
+FROM centos:7
+
+LABEL maintainer="yee.yi@vesoft.com"
+
+RUN yum update -y \
+  && yum install -y centos-release-scl \
+  && yum install -y devtoolset-8 \
+  && yum install -y \
+    autoconf \
+    autoconf-archive \
+    automake \
+    file \
+    git \
+    java-1.8.0-openjdk \
+    libtool \
+    make \
+    maven \
+    ncurses-devel \
+    perl \
+    perl-WWW-Curl \
+    readline-devel \
+    rpm-build \
+    unzip \
+    vim \
+    wget \
+    xz-devel \
+  && yum clean all \
+  && rm -rf /var/cache/yum
+
+RUN scl enable devtoolset-8 -- bash \
+  && mkdir -p /home/nebula \
+  && ln -snf /opt/rh/devtoolset-8/enable /etc/profile.d/devtoolset-8.sh
```

Comment:
How about changing the target name to `devtoolset-8-setup.sh` or other meaningful names?